onnxruntime/core/graph/graph.cc

```diff
@@ -48,22 +48,6 @@ namespace onnxruntime {
     GraphProtoSyncNeeded(sync_needed);               \
   } while (0)
 
-static bool UsingLatestOnnxOpset(const DomainToVersionMap& opset_versions) {
```

Comment:
Earlier there was a comment on why this was deleted. Has it been addressed?

Reply:
It's deleted because it's no longer used. The previous policy of changing strictness based on op set version has been removed entirely, as noted in PR description.

I can't find the earlier comment, so please restate the concern or link to the comment if it hasn't been addressed.